test/atexit.jl

```diff
@@ -0,0 +1,855 @@
+using Test
+
+@testset "atexit.jl" begin
+    function _atexit_tests_gen_cmd_eval(expr::String)
+        cmd_eval = ```
+        $(Base.julia_cmd()) --check-bounds=yes --code-coverage=all
```

Comment:
where are all these flags coming from? It should just be `$(Base.julia_cmd()) -e $expr`

Reply:
Done. I've removed those flags.